harness/determined/cli/experiment.py

```diff
@@ -670,7 +672,9 @@ def kill_experiment(args: Namespace) -> None:
 
 @authentication.required
 def wait(args: Namespace) -> None:
-    exp = client.ExperimentReference(args.experiment_id, cli.setup_session(args))
+    sess = cli.setup_session(args)
+    resp = bindings.get_GetExperiment(session=sess, experimentId=args.experiment_id)
+    exp = client.ExperimentReference._from_bindings(resp.experiment, sess)
```

Comment:
i thought about this, but it would require instantiating an SDK client and changing some of the auth logic in the CLI. i don't know if it's worth doing if we're not going to change all other methods in the cli to use the SDK, which i don't think we should do since often the request and return values are different.